### DISCUSSION TITLE
FluentDivider not working whenever it is used within a Stack?

### DISCUSSION THREAD

**Discussion by PascalVorwerk** [microsoft/fluentui-blazor#2834] [Q&A]
Just wondering if this is known thing, or just something i have been doing wrong. Whenever i pull the FluentDivider out of my stacks it works, but if it's inside a Stack, it won't actually do anything visual. You can still see the component is there, but it just doesn't show a line. If needed, i can supply some code and screenshots after the weekend.

**Comment by vnbaaij** [comment]
Try if giving it a fixed height helps

**Reply by dontpanic21** [reply]
I'm having the same issue, but I can't set a fixed height because I am using FluentAccordian components on either side of the separator and the height is variable (I have set <FluentDivider Style="height: 100%;" with no success). Any suggestions for this case?

**Reply by dvoituron** [reply]
Because **FluentStack** uses `display: flex`, you must specify the **height** and **width** of the **FluentDivider** (with fixed/pixels values, not %).

**Reply by dontpanic21** [reply]
would it be possible to use the calc() function of CSS to calculate the pixels and @bind that to the width and height?

**Reply by dvoituron** [reply]
Can you try that? :-)
